src/client/Container/ContainerDefinition.ts

```diff
@@ -12,4 +12,6 @@ export interface ContainerDefinition {
   defaultTtl?: number;
   /** The conflict resolution policy used to resolve conflicts in a container. */
   conflictResolutionPolicy?: ConflictResolutionPolicy;
+  /** The opearation log policy for a container. */
```

Comment:
nit: this isn't a descriptive comment, imo. Something like "The operation log policy for a container determines the log retention period for your change feed" (not sure if that's correct)